### ISSUE TITLE
Accessibility section missing in schema reference

### ISSUE THREAD

**Issue by dan-odsc** [openreferral/specification#407]
This need to be added followed by a response/thanks to Devin - https://forum.openreferral.org/t/3-0-for-review-and-comment/364/17?u=dan-ods

Holding off on this as we have the response to Marion outstanding. We should respond to that at the same time as we post to Marion.
